tests/sampling/test_forward.py

```diff
@@ -862,11 +862,18 @@ def test_logging_sampled_basic_rvs_prior(self, caplog):
         assert caplog.record_tuples == [("pymc.sampling.forward", logging.INFO, "Sampling: [x, z]")]
         caplog.clear()
 
+        # RV only
         with m:
             pm.sample_prior_predictive(draws=1, var_names=["x"])
         assert caplog.record_tuples == [("pymc.sampling.forward", logging.INFO, "Sampling: [x]")]
         caplog.clear()
 
+        # observed only
+        with m:
+            pm.sample_prior_predictive(draws=1, var_names=["z"])
+        assert caplog.record_tuples == [("pymc.sampling.forward", logging.INFO, "Sampling: [z]")]
```

Comment:
We want this to show [x, z], because x is being sampled as well in order to get z

Reply:
The new test is actually useful to cement what the behavior should be